packages/cli/test/integration/generators/app.integration.js

```diff
@@ -295,7 +295,12 @@ describe('app-generator with tilde project path', () => {
     // eslint-disable-next-line no-invalid-this
     this.timeout(30 * 1000);
 
-    process.chdir(sandbox);
+    // If the test runs outside $home directory
+    if (process.env.CI && !process.env.DEBUG) {
+      process.chdir(sandbox);
+    } else {
+      process.chdir(rootDir);
+    }
```

Comment:
@agnes512 Does this look reasonable?  I had to do this for CI test but I'm not sure how that works.  I've also dropped a clause to compare the sandbox path from L266.

Reply:
Hmm, I find this suspicious. On the next line, we are running a cleanup script in the `sandbox` directory:

https://github.com/strongloop/loopback-next/blob/9aba52e59c80830dd0871c4a087b90ec2aa21db2/packages/cli/test/integration/generators/app.integration.js#L304

Does the behavior of `run-clean` depend on the current working directory? If it does not, then perhaps we can get rid of this first `process.chdir` entirely, and keep the last `chdir(cwd)` only?

```ts
   after(function() {
    // Increase the timeout to accommodate slow CI build machines
    // eslint-disable-next-line no-invalid-this
    this.timeout(30 * 1000);

    build.clean(['node', 'run-clean', sandbox]);
    process.chdir(cwd);
  });
```

Reply:
This appears to be a special case for tilde check. We are using sandbox that is outside the code's project root. Not setting the `process.chdir` gives for example: `Skipping /home/dev/workspace/loopback-next/sandbox/tilde-path-app as it is not inside the project root directory.`